package.json

```diff
@@ -8,8 +8,10 @@
     "dev": "concurrently \"npm run server\" \"npm run client\"",
     "client": "npm run start --prefix client",
     "server": "nodemon server.js",
-    "test": "NODE_ENV=test mocha tests -r chai/register-expect",
-    "test-watch": "nodemon --exec 'npm test'"
+    "test": "concurrently \"npm run test-server\" \"npm run test-client\"",
+    "test-server": "NODE_ENV=test mocha tests -r chai/register-expect",
```

Comment:
install cross-env ...
npm install --save-dev cross-env
change to "test-server": "cross- env NODE_ENV=test mocha tests -r chai/register-expect"